packages/plugin-loyalties-api/src/models/ScoreCampaigns.ts

```
@@ -350,6 +366,49 @@
+                : customFieldData
+          );
+
+          const preparedCustomFieldsData = await sendCoreMessage({
+            subdomain,
+            action: "fields.prepareCustomFieldsData",
+            data: updatedCustomFieldsData,
+            defaultValue: [],
+            isRPC: true,
+          });
+
+          this.updateOwnerScore({
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)